lua/entities/grav_blaster_burst.lua

```diff
@@ -0,0 +1,142 @@
+ENT.Type = "point"
+ENT.Base = "base_point"
+ENT.Spawnable = false
+
+local BURST_LIFETIME_SECS = 3
+local BURST_SPEED_UNITS = 4140 -- In the direction of the burst each frame
+local BURST_EXPLOSION_RADIUS_UNITS = 290
+local BURST_PASSIVE_RADIUS_UNITS = 120 -- Used for passively pulling objects towards the burst as it's flying
+local BURST_DAMAGE_HP = 70 -- Scaled by distance from explosion center
+local BURST_TOSS_IMPULSE_MAGNITUDE = 62200 -- Hammer units
+local BURST_PASSIVE_IMPULSE_MAGNITUDE = BURST_TOSS_IMPULSE_MAGNITUDE / 2 -- Hammer units, scaled down because it's passive (during flight)
+local BURST_PLAYER_FORCE_MULTIPLIER = 0.4 -- Players weigh ~170lb, 85 kg and this is as about as heavy as a small desk in hl2 which can easily be tossed around. Basically the weight scale in HL2 is weird.
```

Comment:
Another more general point on these constants, I think a lot of these should actually just be properties on the entity instance (like magnitude and speed) which are then defaulted to these values from the swep file

Doing this allows for easily increasing the functionality of the weapon by e.g. adding a charge time to increase the power and/or speed of the bursts